src/goInstallTools.ts

```diff
@@ -129,6 +129,11 @@ export function installTools(missing: Tool[], goVersion: GoVersion): Promise<voi
 	if (!fs.existsSync(toolsTmpDir)) {
 		fs.mkdirSync(toolsTmpDir);
 	}
+	// Write a temporary go.mod file to support Go 1.11.
```

Comment:
If this is needed only for 1.11 and not newer versions, should we perhaps add a check for the Go version?

Reply:
It shouldn't be a problem for the later Go versions, but I added a check anyway.